README.md

```diff
@@ -172,6 +173,36 @@ In the case of an anonymous rest parameter, simply omit the name:
 (...: [...Any]) => Array
 ```
 
+### Type Variables
+
+Type variables are types that do not need to be declared in advance. They may represent any type, but a single type variable may only represent one type at a time in the scope of the signature being declared.
+
+The signature for double is usually thought of like this:
+
+```js
+double(x: Number) => Number
+```
+
+But what if we want it to accept objects as well?
+
+```js
+const one = {
+  name: 'One',
+  valueOf: () => 1
+};
+
+double(one); // 2
+```
+
+In that case, we'll need to change the signature to use a type variable:
+
+```js
+double(x: n) => Number
```

Comment:
The problem that you have here is that someone could get confused over the next paragraph. You are using a single letter twice: it might induce a double take for someone unfamiliar with the concept.